### PR TITLE
Change user_url parsing

### PR DESCRIPTION
**Pull request by genevera**
Pick up the RSS url and chop off /rss.

### CODE REVIEW

**Comment by genevera** [comment]
Yep - works for me locally!

image

File: Extensions/xkit_patches.js [hunk 1]
```diff
@@ -2182,8 +2182,8 @@ XKit.extensions.xkit_patches = new Object({
 					m_return.likes = true;
 				}
 
-				if ($('meta[name="twitter:title"]').length) {
-					m_return.user_url = $('meta[name="twitter:title"]').attr("content");
+				if ($('link[type="application/rss+xml"]').length) {
+					m_return.user_url = $('link[type="application/rss+xml"]').attr("href").replace(/^(.*)\/rss.*$/, function(match, $1) {return $1;});
```
Comment (hobinjk) [inline Extensions/xkit_patches.js]: Good find! Hopefully Staff will keep this one around!

The `.replace` would be simpler to understand as `.replace(/\/rss.*/, '')`. The approach you're currently taking is a bit overkill for chopping off everything including and following '/rss'.

**Comment by genevera** [comment]
@nightpool ping

**Comment by nightpool** [comment]
@genevera looks like @hobinjk left a review. should be g2g after that gets fixed.

**Comment by genevera** [comment]
Awesome! Thanks.  Will sort that out this weekend.

File: Extensions/xkit_patches.js (hunk 1 above)
Comment (genevera) [inline Extensions/xkit_patches.js]: @hobinjk done!

**Comment by genevera** [comment]
@hobinjk @nightpool just applied the suggested change.

**Comment by genevera** [comment]
Also, this only works on the dashboard.   It could be made to work everywhere the user menu shows up, I think.

**Comment by hobinjk** [comment]
Oh, sorry! I meant that you could replace
```javascript
replace(/^(.*)\/rss.*$/, function(match, $1) {return $1;});
```
with
```javascript
replace(/\/rss.*$/, '');
```

**Comment by genevera** [comment]
Oh God duh - of course there's no need for the grouping part.

**Comment by genevera** [comment]
@hobinjk made the requested changes

**Comment by genevera** [comment]
@nightpool @hobinjk ping :)

**Comment by hobinjk** [comment]
@genevera Tested locally and LGTM, nice work! There's just one last change to make. For XKit to realize that it needs to update xkit_patches.js the version number at the beginning of the file needs to be incremented. It's currently `6.8.0`, for this type of bug fix the version should be changed to 6.8.1.

**Comment by genevera** [comment]
@hobinjk thank you!  version is bumped 🎉

**Comment by hobinjk** [comment]
@homu r+
🎉 ✨ ✨ ✨

**Comment by genevera** [comment]
:woman_cartwheeling:

**Pull request merged**